packages/sg1/src/lib.rs

```diff
@@ -40,10 +40,7 @@ pub fn fair_burn(fee: u128, developer: Option<Addr>, res: &mut Response) {
             }));
             event = event.add_attribute("dev", dev.to_string());
             event = event.add_attribute("dev_amount", Uint128::from(dev_fee).to_string());
-            (
-                Decimal::percent(FEE_BURN_PERCENT - DEV_INCENTIVE_PERCENT),
-                dev_fee,
-            )
+            (Decimal::percent(FEE_BURN_PERCENT), dev_fee)
```

Comment:
you could pull `FEE_BURN_PERCENT` out of this `match developer` block since it is not used there